applications/StructuralMechanicsApplication/tests/test_StructuralMechanicsApplication.py

```diff
@@ -359,5 +361,21 @@ def AssembleTestSuites():
 
 
 if __name__ == '__main__':
-    KratosUnittest.runTests(AssembleTestSuites())
+    print("\nRunning cpp unit tests ...")
     run_cpp_unit_tests.run()
+    print("Finished running cpp unit tests!")
+
+    print("\nRunning mpi python tests ...")
+    try:
+        import KratosMultiphysics.mpi as KratosMPI
+        import KratosMultiphysics.MetisApplication as MetisApplication
+        import KratosMultiphysics.TrilinosApplication as TrilinosApplication
+        p = Popen(["mpiexec", "-np", "2", "python3", "test_StructuralMechanicsApplication_mpi.py"])
```

Comment:
@roigcarlo are you fine with what I am proposing here?
I think it is very convenient to have everything executed from one file bcs otherwise we will forget to run the MPI tests